src/test_helpers/component.cljs

```diff
@@ -30,10 +30,18 @@
   [text]
   (rtl/screen.getByText text))
 
+(defn find-by-text
+  [text]
+  (rtl/screen.findByText text))
+
 (defn get-by-label-text
   [label]
   (rtl/screen.getByLabelText (name label)))
 
+(defn query-by-label-text
```

Comment:
I added this function to successfully test when a component is not rendered because if we call `get-by-label-text` we get an exception and the test fails.

Reply:
nice one @ulisesmac!